.github/workflows/release.yaml

```diff
@@ -38,14 +38,27 @@ jobs:
             - name: Install Dependencies
               run: pnpm install
 
-            - name: Create Release Pull Request
+            - name: Build
+              run: pnpm build
+
+            - name: Test
+              run: pnpm test
```

Comment:
This should be test:coverage só that the next step has the data.

Have to check if there's a vitest config at this level that generates the json coverage report